canvas_modules/common-canvas/src/common-canvas/cc-bottom-panel.jsx

```diff
@@ -27,14 +27,17 @@ class CanvasBottomPanel extends React.Component {
 			panelHeight: 393
 		};
 
-
 		this.logger = new Logger("CC-Bottom-Panel");
 
 		this.onMouseUp = this.onMouseUp.bind(this);
 		this.onMouseDown = this.onMouseDown.bind(this);
 		this.onMouseMoveY = this.onMouseMoveY.bind(this);
 	}
 
+	componentWillUnmount() {
+		document.removeEventListener("mouseup", this.onMouseUp, false);
+	}
+
```

Comment:
See my comment about the addEventListener.  There is a better way to handle the addition and removal of the mouseUp event listener so that it is only in existence while it is needed. Let's discuss if necessary.